.github/workflows/wfc_lifecycle.yml

```diff
@@ -66,11 +66,32 @@ on:
       private_key:
         required: false
 jobs:
+  branches:
+    if: github.event.workflow || github.event.schedule
+    uses: ./.github/workflows/wfc_active_branches.yml
+  harden_security:
+    needs: [branches]
+    name: Harden Security
```

Comment:
Should we also metion that we ensure actions use pinned git SHAs?

Reply:
I think there's a step that's named doing just this